More visibility to SmallRye

radcortez opened the pull request.

radcortez committed:
More visibility to SmallRye.

gastaldi reviewed (approved):
LGTM

gsmet reviewed (dismissed):
What's the rationale of this? I read a lot of things about people being happy to have Quarkus compatible with MicroProfile. Will it be visible enough after this change?

Also, you will need to install redirects in the website.

kenfinnigan commented on docs/src/main/asciidoc/config-reference.adoc:
```diff
@@ -13,10 +13,9 @@ include::./attributes.adoc[]
-Moreover, there are several additional features which are made available by the SmallRye Config project as well as by Quarkus itself.
+A Quarkus application and Quarkus itself (core and extensions) are both configured via the same mechanism that leverages
+the https://github.com/smallrye/smallrye-config[SmallRye Config] API an implementation of the MicroProfile Config
+specification. Moreover, there are several additional features which are made available by Quarkus itself.
```
Would be good to link to the MP Config spec like was done before

kenfinnigan commented on docs/src/main/asciidoc/micrometer.adoc:
```diff
@@ -402,9 +402,9 @@ to inject any configuration attributes you need to configure the registry. Most
 like `quarkus-micrometer-registry-statsd`, define a producer for registry-specific configuration objects
 that are integrated with the Quarkus configuration model.
 
-== Support for the MicroProfile Metrics API
```
These changes should be reverted.

This section is specifically about enabling the use of MP Metrics APIs with Micrometer, nothing to do with SmallRye Metrics.

cc @ebullient

kenfinnigan commented:
@gsmet There was a discussion yesterday around making it clearer that Quarkus is using SmallRye projects, which does implement MicroProfile specifications, to make it easier to highlight when SmallRye goes beyond a specification

gsmet commented:
There are now conflicts due to the master -> main change.

radcortez commented:
> There are now conflicts due to the master -> main change.

Fixed.

Also added the redirects in https://github.com/quarkusio/quarkusio.github.io/pull/925. Do we need to add an initial version of the renamed guides or everything is synced automatically?

gsmet commented:
@radcortez no, they will be added automatically. I just need to think about merging the PR after the release :).

@kenfinnigan could you have another look?

radcortez commented:
Thanks!
